Initial Commit

damien-murphy opened the pull request:
Initial Commit

GauSim commented on qualtrics/backend/Dockerfile:
```diff
@@ -0,0 +1,12 @@
+FROM registry.dev.coresuite.com/docker-base-webapp-nodejs-nginx:1.0.2
```
This docker file is not public, please use a public docker file as vase

GauSim commented on s4hana/backend/Dockerfile:
```diff
@@ -0,0 +1,12 @@
+FROM registry.dev.coresuite.com/docker-base-webapp-nodejs-nginx:1.0.2
```
Same here, use different basr

damien-murphy commented:
Added Angular frontend code and removed backend server side rendered views.

See the widget hosted here: https://widget-frontend.sc-xf-fsm-dev.cluster.extend.sap.cx
